### ISSUE TITLE
New WCAG Criteria -  2.4.13 Focus Appearance - AAA

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/nrrd#3048]
We know we have some missing visible indicators, but we also need to check some for this criteria: is at least as large as the area of a 2 CSS pixel thick perimeter of the unfocused component or sub-component, and has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states.

**Comment by Maroyafaied** [comment]
Manually check the width and contrast of focus indicators

**Comment by lpgoldstein** [comment]
No longer relevant
